Inception/ClientApp/src/App/Home/HomeComponent.ts

```diff
@@ -1,15 +1,29 @@
 import { OnInit, Component } from "@angular/core";
+import { Service } from "../Services";
+import { SiteTestResult } from "../Models";
 
 @Component(
```

Comment:
"(" and ")" should on separate lines with single indent

Reply:
+